src/checkable_continuous_flyout.js

```diff
@@ -103,6 +103,21 @@ export class CheckableContinuousFlyout extends ContinuousFlyout {
     this.checkboxes_.clear();
   }
 
+  layout_(contents, gaps) {
+    super.layout_(contents, gaps);
+    // We want large gaps between categories (see GAP_Y), but don't want those
+    // counted as part of the category for purposes of scrolling to show the
+    // category, so we reset/adjust the label gaps used for the scroll position
+    // calculation here.
+    this.labelGaps.fill(4);
```

Comment:
Express these in terms of GRID_UNIT if possible (here and below)

Reply:
Done